### PR TITLE
[TT-26] Restructure of integration-tests

### PR DESCRIPTION
**Pull request by smickovskid**
- Removed ginkgo
- Refactored structure

### CODE REVIEW

**Review by sdrug — DISMISSED** [review]
LGTM! Let's also update the README since ginkgo suite is not used anymore.

**Review by tateexon — DISMISSED** [review]

File: .tool-versions [hunk 1]
```diff
@@ -4,15 +4,14 @@ golang 1.20.1
 python 3.9.13
 
 # Tools
-ginkgo 2.1.3
```
Comment (aalu1418) [inline .tool-versions:7]: 🎉

File: integration-tests/go.mod [hunk 2]
```diff
@@ -4,16 +4,22 @@ go 1.19
...
+
+require (
+	github.com/smartcontractkit/ocr2keepers v0.6.6 // indirect
+	github.com/stretchr/objx v0.5.0 // indirect
+	github.com/umbracle/ethgo v0.1.3 // indirect
+	github.com/umbracle/fastrlp v0.0.0-20220527094140-59d5dd30e722 // indirect
+	github.com/valyala/fastjson v1.4.1 // indirect
 )
 
 require (
```
Comment (aalu1418) [inline integration-tests/go.mod]: can we combine the two `// indirect` sections to keep it cleaner? (not sure why go decides to split things)

**Review by aalu1418 — CHANGES_REQUESTED** [review]
🎉 

just one small change to keep the `go.mod` a little cleaner :)

**Review by sdrug — DISMISSED** [review]

**Review by aalu1418 — DISMISSED** [review]

**Review by sdrug — APPROVED** [review]

**Pull request merged**